src/Components/Testimonial.js

```diff
@@ -1,13 +1,121 @@
 import React, { Component } from "react";
 import TestimonialCard from "./TestimonialCard";
+import "tachyons";
+import Slider from "react-slick";
 
-export default class Testimonial extends Component {
+class Testimonial extends Component {
   render() {
+    const settings = {
+      dots: true,
+      className: "centered",
+      lazyLoad: true,
+      centerMode: true,
+      infinite: true,
+      centerPadding: "30px",
+      slidesToShow: 3, 
+      speed: 500,
+      responsive: [
+        {
+          breakpoint: 1024,
+          settings: {
+            slidesToShow: 3,
+            infinite: true,
+            dots: true
+          }
+        },
+        {
+          breakpoint: 900,
+          settings: {
+            slidesToShow: 1
+          }
+        },
+        {
+          breakpoint: 480,
+          settings: {
+            slidesToShow: 1
+          }
+        }
+      ],
+    };
+
+    const h1_style = {
+      letterSpacing: "0px",
+      fontSize: "2.3rem",
+      fontWeight: "390",
+      fontFamily: "Arial",
+      textAlign: "center",
+    };
+    
     return (
       <div>
-        <TestimonialCard />
-        <TestimonialCard />
+        <h1 className="mt6"  style={h1_style}>What Our <span style={{color: '#008dc8'}}>Students </span>Say?</h1>
+        <br />
+        <Slider {...settings}>
+          <div>
+            <TestimonialCard
```

Comment:
This piece of code can be written in a better way. As I can see the following component is repeated multiple times:

```js
<div>
        <TestimonialCard
              test={{
                name: "Sherley Williams",
                imgUrl: "https://picsum.photos/200",
                detail1:
                  "Lorem ipsum dolor sit amet consectetur adipisicing elit. Est quaerat tempora, voluptatum quas facere dolorum aut cumque nihil nulla harum nemo distinctio quam blanditiis dignissimos"
              }}
            />
  </div>
```

Better way of doing it would be running a map over a JSON containing `name`, `imgUrl` and `detail1`.

CC: @Manvityagi